firmware/common/libs/timer/oscc_timer.cpp

```diff
@@ -10,7 +10,9 @@
 
 
 static void (*timer_1_isr)(void);
+#ifndef __AVR_ATmega32U4__
```

Comment:
Note for posterity: this might be worth thinking about more in case we have troubles with the petrol brake actuator, but off the cuff it looks like this concern probably won't ever come up.

Reply:
That's an Atmega2560 so an __AVR_ATmega2560__ would be the pre processor for that board.